misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobQueue.kt

```diff
@@ -129,7 +129,8 @@ internal class SqsJobQueue @Inject internal constructor(
     .withStringValue(this)
 
   private fun getDelaySeconds(deliveryDelay: Duration?): Int {
-    return (deliveryDelay?.toMillis() ?: 0 / 1000).toInt()
+    val delayMillis = deliveryDelay?.toMillis() ?: 0
```

Comment:
Should we have a test for this?

Reply:
I'm sure one is useful - I was just biasing to getting something fixed asap.

Reply:
Agree, let's get this out there and then reach out to the author of the breaking change to add a test.